Scripts/BuildPhases/ApplyConfiguration.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/sh
 
+set -e
+
```

Comment:
What about the more common way of setting this on the shebang directly, and add other common options for super-safe mode?

```suggestion
#!/bin/sh -euo pipefail
```

Reply:
I had never seen this, but I like it.

I'm curious to know what makes you say that it is more common? That's not to challenge your statement, but since I'd never seen that syntax, I'm curious to know which authoritative resources I can look into next time 😄 Thanks.

Reply:
Ahah i don't have any authoritative source on that, it's just that my experience is opposite to you, as I saw the flags set in the shebang way more often that seeing them set using `set` via a separate command, so I assumed the former was more common 😛

I've often seen, in articles suggesting to set those flags in eg Script Build Phases — to ensure the Xcode build fails as soon as one command in the script fails instead of only looking at exit code of the last command of the script — suggestions to set it on the "Shell" field (eg `/bin/sh -e`) too, so that's what I got used to do… but YMMV 🙃

Reply:
Thanks for the extra context. I like this setup and I'm keen to use it. 

I tried researching whether it was a appropriate but didn't get far, too much conflicting information.

I say let's adopt it (it's neat to have them all in a oneliner) and see how it plays out for us.

Reply:
Well this thread answers _that_ question for me 😜